spec/lib/data_update_scripts/populate_explicit_follow_points_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+require Rails.root.join(
+  "lib/data_update_scripts/20201120001727_populate_explicit_follow_points.rb",
+)
+
+describe DataUpdateScripts::PopulateExplicitFollowPoints do
+  it "updates follows that had points to having explicit points", :aggregate_failures do
+    follow = create(:follow, points: 3)
+    expect(follow.explicit_points).to eq(1.0)
```

Comment:
**suggestion (non-blocking):** This could be more idiomatically expressed with a `change` assertion:

```ruby
expect do
  described_class.new.run
end.to change { follow.reload.explicit_points }.from(1.0).to(3)
```